Shared/R4/files/BasicClinicalInfoPrepopulation-0.1.0.cql

```diff
@@ -0,0 +1,31 @@
+library BasicClinicalInfoPrepopulation version '0.1.0'
+using FHIR version '4.0.0'
+include FHIRHelpers version '4.0.0' called FHIRHelpers
+
+context Patient
+
+define Today: Today()
+
+define AnswerYes:
```

Comment:
This statement doesn't seem to be used at all. It could be removed.

Side note: This is not a proper way to declare a code to be used. They should be declared at the top of the file using a `code` statement. https://cql.hl7.org/02-authorsguide.html#code

Reply:
fixed